owncloudApp/src/androidTest/java/com/owncloud/android/settings/logs/OCSettingsLogTest.kt

```diff
@@ -1,65 +0,0 @@
-/**
- * ownCloud Android client application
- *
- * @author Jesús Recio @jesmrec
- * Copyright (C) 2020 ownCloud GmbH.
- * <p>
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2,
- * as published by the Free Software Foundation.
- * <p>
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- * <p>
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-package com.owncloud.android.settings.logs
-
-import androidx.test.espresso.Espresso.onView
-import androidx.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu
-import androidx.test.espresso.assertion.ViewAssertions.matches
-import androidx.test.espresso.matcher.ViewMatchers.isDescendantOfA
-import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
-import androidx.test.espresso.matcher.ViewMatchers.withId
-import androidx.test.espresso.matcher.ViewMatchers.withText
-import androidx.test.platform.app.InstrumentationRegistry
-import androidx.test.rule.ActivityTestRule
-import com.owncloud.android.R
-import com.owncloud.android.ui.activity.LogHistoryActivity
-import org.hamcrest.Matchers.allOf
-import org.junit.Rule
-import org.junit.Test
-
-class OCSettingsLogTest {
```

Comment:
This one won't be removed either. In fact, in the new branches it has changed its name to `LogHistoryActivity`, but it will remain the same with some little refactor to remove deprecated code.